### ISSUE TITLE
Proper selection of approved and denied word in kubectl output

### ISSUE THREAD

**Issue by pandeyvinod** [kubernetes/kubernetes#91876]
**What happened**: When you denied any csr request through kubectl. it says approved in output though its denied when you checked in status

**What you expected to happen**:
It need to Denied in output itself.

**How to reproduce it (as minimally and precisely as possible)**:
1- request csr for any user
2- try to deny that request

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`): 1.18
- Cloud provider or hardware configuration: cloud provider
- OS (e.g: `cat /etc/os-release`): ubuntu
- Kernel (e.g. `uname -a`): 4.4
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:

**Comment by knight42** [comment]
@pandeyvinod I cannot reproduce this problem with the following steps, could you post the commands you execute?

1. create csr request
```
$ cat <<EOF | cfssl genkey - | cfssljson -bare server
{
  "hosts": [
    "my-svc.my-namespace.svc.cluster.local",
    "my-pod.my-namespace.pod.cluster.local",
    "192.0.2.24",
    "10.0.34.2"
  ],
  "CN": "my-pod.my-namespace.pod.cluster.local",
  "key": {
    "algo": "ecdsa",
    "size": 256
  }
}
EOF
```

```
$ cat <<EOF | kubectl apply -f -
apiVersion: certificates.k8s.io/v1beta1
kind: CertificateSigningRequest
metadata:
  name: my-svc.my-namespace
spec:
  request: $(cat server.csr | base64 | tr -d '\n')
  usages:
  - digital signature
  - key encipherment
  - server auth
EOF
```

```
$ kubectl get csr
NAME                  AGE   REQUESTOR   CONDITION
my-svc.my-namespace   7s    admin       Pending
```

2. deny csr request
```
$ kubectl certificate deny my-svc.my-namespace
certificatesigningrequest.certificates.k8s.io/my-svc.my-namespace denied

$ kubectl version
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.3", GitCommit:"2e7996e3e2712684bc73f0dec0200d64eec7fe40", GitTreeState:"clean", BuildDate:"2020-05-21T14:51:23Z", GoVersion:"go1.14.3", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.7", GitCommit:"be3d344ed06bff7a4fc60656200a93c74f31f9a4", GitTreeState:"clean", BuildDate:"2020-02-11T19:24:46Z", GoVersion:"go1.13.6", Compiler:"gc", Platform:"linux/amd64"}
```

looks like it works as expected

**Comment by liggitt** [comment]
I think this is a duplicate of https://github.com/kubernetes/kubernetes/issues/88937 and was fixed in 1.17 in https://github.com/kubernetes/kubernetes/pull/84400

with kubectl 1.18.0:

```sh
kubectl version
```

> ```
> Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.0", GitCommit:"9e991415386e4cf155a24b1da15becaa390438d8", GitTreeState:"clean", BuildDate:"2020-06-09T19:43:51Z", GoVersion:"go1.13.4", Compiler:"gc", Platform:"darwin/amd64"}
> Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.0", GitCommit:"9e991415386e4cf155a24b1da15becaa390438d8", GitTreeState:"clean", BuildDate:"2020-06-09T19:43:51Z", GoVersion:"go1.13.4", Compiler:"gc", Platform:"darwin/amd64"}
> ```

```sh
kubectl create -f hack/testdata/csr.yml 
```

> ```
> certificatesigningrequest.certificates.k8s.io/foo created
> ```
 
```sh
kubectl get csr
```

> ```
> NAME   AGE   SIGNERNAME                     REQUESTOR      CONDITION
> foo    9s    kubernetes.io/legacy-unknown   system:admin   Pending
> ```

```sh
kubectl certificate deny foo
```

> ```
> certificatesigningrequest.certificates.k8s.io/foo denied
> ```

```sh
kubectl get csr
```

> ```
> NAME   AGE   SIGNERNAME                     REQUESTOR      CONDITION
> foo    30s   kubernetes.io/legacy-unknown   system:admin   Denied
> ```

/close

**Comment by pandeyvinod** [comment]
some how i mentioned wrong version in description. correct version was 1.16 as mentioned above.

Looks like its already fixed in the newest release.
